core/tx_pool.go

```diff
@@ -1574,10 +1576,10 @@ func ValidateTransactorBalanceCoversTx(tx *types.Transaction, from common.Addres
 			return err
 		}
 
-		// To match the logic in canPayFee() state_transition.go, we require the balance to be strictly greater than the fee,
-		// which means we reject the transaction if balance <= fee
+		// To match the logic in canPayFee() state_transition.go, we require the balance to be greater than or equal to the fee,
```

Comment:
Could you add comment about the HF behavior here?

Reply:
Oops, I misunderstood your prior comment.